simulators/eth2/testnet/scenarios.go

```diff
@@ -39,12 +39,12 @@ func Phase0Testnet(t *hivesim.T, env *testEnv, n node) {
 		MergeForkEpoch:          20,
 		ValidatorCount:          VALIDATOR_COUNT,
 		SlotTime:                SLOT_TIME,
-		TerminalTotalDifficulty: TERMINCAL_TOTAL_DIFFICULTY,
+		TerminalTotalDifficulty: TERMINRAL_TOTAL_DIFFICULTY,
```

Comment:
```suggestion
		TerminalTotalDifficulty: TERMINAL_TOTAL_DIFFICULTY,
```

Reply:
Ha - sorry, went around globally replacing some things and apparently this was a casualty.